pkg/manifest/manifest.go

```diff
@@ -393,6 +400,9 @@ type Step struct {
 }
 
 func (s *Step) Validate(m *Manifest) error {
+	if s == nil {
+		return errors.New("found an empty step. probably you forgot to mention it")
```

Comment:
I might suggest we just leave the error message as a succinct `found an empty step`.

Reply:
Agreed, let's use Vaughn's suggested wording for the error message, I think it is more clear